src/models/Review.java

```diff
@@ -44,8 +49,16 @@ public int getMAX_REVIEW_LENGTH() {
 	}
 	
 	// checks if the review is longer than the maximum review length
-	public boolean isReviewTooLong() {
+	public boolean isReviewTooLong(String review) {
```

Comment:
If we're going to make it take a string as a param, we should make the method static and have it be `return review.length() >= this.getMAX_REVIEW_LENGTH();`